buildmaster/ATLAS_1JET_8TEV_R06/metadata.yaml

```
@@ -0,0 +1,74 @@
+# Created on Mar 2023, author: Mark N. Costantini
+
+# Generalia
+setname: "ATLAS_1JET_8TEV_R06"
+
+version: 0
```

```suggestion
version: 1
```

This is just a convention ofc